modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -28,6 +28,15 @@ definition {
 				userEmailAddress = "partner@liferay.com");
 		}
 
+		task ("Add Maneger Channel role to the user") {
```

Comment:
Hi @mateussandes23, just a typo here. Could you please replace 'Maneger' for 'Manager'? Thanks!